### DISCUSSION TITLE
How can I ensure that storage components are successfuly deployed?

### DISCUSSION THREAD

**Discussion by Douglas235** [clusterpedia-io/clusterpedia#536] [Q&A]
The success or failure will be known after the component is deplyed

**Comment by Iceber** [comment]
If the storage component fails to deploy, clusterpedia fails to initialize the storage layer.

In [helm chart](https://github.com/clusterpedia-io/clusterpedia-helm/tree/main/charts/clusterpedia) there is also an init container that checks in advance if the storage component is available
